pkg/controller/status/status.go

```diff
@@ -78,7 +78,8 @@ type statusManager struct {
 	statefulsets map[string]types.NamespacedName
 	cronjobs     map[string]types.NamespacedName
 	lock         sync.Mutex
-	enabled      bool
+	crQueried    bool
```

Comment:
I've looked through this a few times but I'm still not seeing the importance of knowing if the CR was queried or not. What is this flag supposed to indicate?

Reply:
It lets us know that we have at least Queried for the CR (because OnCRFound or OnCRNotFound has been called at least once). This prevents us from removing the tigerastatus when we haven't tried querying the CR yet.

Reply:
Previously we would just remove the tigerastatus if enabled was false and it was false at startup. So to make sure we only remove it after we've queried it I added this flag.